scripts/ios/build-sargon.sh

```diff
@@ -45,18 +45,6 @@ generate_ffi() {
     local TARGET_FOR_DYLIB_PATH="aarch64-apple-ios"
   fi 
   cargo run --features build-binary --bin sargon-bindgen generate --library target/$TARGET_FOR_DYLIB_PATH/release/lib$1.dylib --language swift --out-dir target/uniffi-xcframework-staging
-
-  if $release; then
-    echo "🚢 Ensuring Sargon build for release - that it will work for e.g. iOS wallet to archive."
-    sed -i '' 's/let useLocalFramework = false/let useLocalFramework = true/' Package.swift
-    swift build -c release || exit $?
```

Comment:
bah. ofc this did not work, I've only built Rust project but not Xcframeworks yet. In fact, safest to not modify the `build` script at all, so I've reverted back to what it was a couple of days ago...